.github/workflows/ci.yml

```diff
@@ -421,7 +421,6 @@ jobs:
       - name: Install HDF5
         run: sudo apt update && sudo apt install -y libhdf5-dev
       - run: pytest modin/test/storage_formats/omnisci/test_internals.py
-      - run: MODIN_BENCHMARK_MODE=True pytest modin/pandas/test/internals/test_benchmark_mode.py
```

Comment:
Why isn't this being tested anymore?

Reply:
This is apparently because although it inherits the pandas partition manager, it does not use methods that are wrapped in the `wait_computations_if_benchmark_mode` decorator. I wonder if he uses at least something from the inherited class. If not, then it might be better to inherit from a more basic object. OK for now.